Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,81 @@
+#include "Person.h"
+#include <iostream>
+#include <string>
+
+using namespace std;
```

Comment:
In general, do not use `using namespace std`. This is considered bad practice because you open it up to ambiguity.
More info:
[https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice](url)